--- clutter-1.6.6/configure.ac.orig	2011-02-21 12:11:32.000000000 +0100
+++ clutter-1.6.6/configure.ac	2011-03-05 20:22:52.847417132 +0100
@@ -1127,7 +1127,7 @@
 
 dnl === Dependencies, compiler flags and linker libraries =====================
 # strip leading space
-BACKEND_PC_FILES=${BACKEND_PC_FILES/# }
+BACKEND_PC_FILES=${BACKEND_PC_FILES# }
 
 CLUTTER_REQUIRES="$CLUTTER_BASE_PC_FILES $IMAGE_PC_FILES $BACKEND_PC_FILES"
 PKG_CHECK_MODULES(CLUTTER_DEPS, [$CLUTTER_REQUIRES])
